src/test/java/hu/oe/nik/szfmv/automatedcar/model/WorldObjectTest.java

```diff
@@ -13,10 +10,4 @@
     public void init() {
         worldObject = new WorldObject(100, 100, "car_2_white.png");
     }
-/*
-    @Test
-    public void addition() {
-        assertEquals(100, worldObject.getPosX());
-    }
-*/
```

Comment:
Ez több okból sem jó. Ha már egyszer volt valamire teszt, akkor azt ne töröljük, ha eltörik, javítsuk. Másrészt, most van egy tesztosztályunk, amit inicializálunk, aztán nem kezdünk vele semmit? Így nem sok értelme van az osztálynak. Persze a dead commentet írtani kell, de ne így.

Reply:
Személy szerint nem szeretem a kikommentelt blokkokat, ha kell megírod újra

Reply:
Na erre inkább nem mondok semmit. Mergeljük akkor, minek is foglalkozzunk a TDD-vel. :) De akkor már adjunk az érzésnek és töröljük az osztályt is, mert ennek így semmi értelme nincs...

Reply:
Nem mintha fontos teszt lenne, csak szebb lett volna javítani, mint törölni. Mondjuk az is érdekelne, hogy került ez be a branchünkre.